projects/hmpps-auth-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/controller/UserController.kt

```diff
@@ -23,24 +23,24 @@ import uk.gov.justice.digital.hmpps.service.UserService
 @Tag(name = "User details")
 class UserController(private val userService: UserService) {
     @GetMapping(value = ["/user/{username}"])
-    @PreAuthorize("hasAnyRole('ROLE_DELIUS_USER_AUTH', 'ROLE_DELIUS_USER_DETAILS')")
+    @PreAuthorize("hasAnyRole('ROLE_DELIUS_USER_AUTH','ROLE_DELIUS_USER_DETAILS','PROBATION_API__HMPPS_AUTH__AUTHENTICATE')")
```

Comment:
Should the endpoints in this file have roles named something other than authenticate, as they're just getting user details?
```suggestion
    @PreAuthorize("hasAnyRole('ROLE_DELIUS_USER_AUTH','ROLE_DELIUS_USER_DETAILS','PROBATION_API__HMPPS_AUTH__USER_DETAILS')")
```